core/src/main/java/org/apache/accumulo/core/file/blockfile/cache/tinylfu/TinyLfuBlockCache.java

```diff
@@ -65,12 +66,14 @@ public final class TinyLfuBlockCache implements BlockCache {
       .createScheduledExecutorService(1, "TinyLfuBlockCacheStatsExecutor", true);
 
   public TinyLfuBlockCache(Configuration conf, CacheType type) {
-    cache = Caffeine.newBuilder()
-        .initialCapacity((int) Math.ceil(1.2 * conf.getMaxSize(type) / conf.getBlockSize()))
-        .weigher((String blockName, Block block) -> {
+    cache = Caches.getInstance().getCache(CacheName.TINYLFU_BLOCK_CACHE,
```

Comment:
There can be multiple instances of the block cache, like an instance for indexes and another for data.  Will these different instances come across in the metrics?

Reply:
Short answer: the block caches won't report via Micrometer

Long answer: The block caches report their stats differently, via a CacheStats object IIRC. Because `recordStats` is called on line 72 below, the `recordStats` call in Caches throws an IllegalStateException and reporting via Micrometer won't occur.

Reply:
I reworked Caches in b663e71. For the block caches, it just returns a builder w/o metrics configuration.